apps/swiss/telnet.c

```diff
@@ -850,7 +850,7 @@ Return Value:
 {
 
     INT Status;
-    ssize_t Written;
```

Comment:
The return type of write() is ssize_t, so I think this was correct as it was before. If you're solving a pointer addition warning, our new macro should help you do: `Buffer = POINTER_ADD(Buffer, Written);`